__tests__/app.test.js

```diff
@@ -0,0 +1,62 @@
+const app = require("../app");
+const request = require("supertest");
+const db = require("../db/connection");
+const seed = require("../db/seeds/seed");
+const data = require("../db/data/test-data");
+
+beforeEach(() => {
+  return seed(data);
+});
+afterAll(() => {
+  db.end();
+});
+
+describe("/api", () => {
```

Comment:
tests looks great. I do wonder if all of the these tests could be refactored into a single test. The way that we have it now, you are making 3 requests to the database for the same information. We could write a single test and have these assertions inside. 

Just make sure that we are testing the length of the array before running the .forEach()

Reply:
Aside from that, everything else is looking good :)